### PR TITLE
Fixes broken cryo on Cargodise Lost, expands areas and APC's

### PR DESCRIPTION
**Pull request by Zergspower**
## About The Pull Request

Recently the cryo computer somehow disappeared from the .dmm and anytime someone tries it floods the logs with errors.

## How This Contributes To The Skyrat Roleplay Experience

The freighter after the big power hammer never really got a pass to assist there, and solars simply aren't really a solution all the time given their rather finicky and terrible generation. Spreading the APC's out prevents the ship de-powering in about 50 seconds while not altering the draw of the ship. 

## Changelog

:cl:
add: Cargodise Lost Ruin: Added new areas and APC's for: Bridge, Kitchen, Trauma, Mining, Utility/power, Hallway and Crew Quarters to the Cargodise Lost Freighter ruin
qol: Cargodise Lost Ruin: Power usage is now spread through multiple APC's for Cargodise
fix: Cargodise Lost Ruin: fixed cryo error spam by returning the cryo-monitor
/:cl:

### CODE REVIEW

File: modular_skyrat/modules/mapping/code/areas/space.dm [hunk 1]
```diff
@@ -285,3 +285,36 @@
 	name = "\improper P-T Solar Array"
 	icon_state = "solarsS"
 	has_gravity = STANDARD_GRAVITY
+
+// Cargodise Lose Freighter
```
Comment (Zonespace27) [inline modular_skyrat/modules/mapping/code/areas/space.dm]:
```suggestion
// Cargodise Lost Freighter
```

**Review by Zonespace27 — DISMISSED** [review]
Your changelog's bad (not actually saying _what's_ being changed)

**Comment by Zergspower** [comment]
> 

I mean, how else would you word it? I made areas, added the cryo machine back and gave said areas APC's

**Comment by Zonespace27** [comment]
> > 
> 
> I mean, how else would you word it? I made areas, added the cryo machine back and gave said areas APC's

the changelog doesn't say where the APCs are being added (to the ruin), where the new areas are (again, in the ruin), etc.

**Comment by Zergspower** [comment]
> > > 
> > 
> > 
> > I mean, how else would you word it? I made areas, added the cryo machine back and gave said areas APC's
> 
> the changelog doesn't say where the APCs are being added (to the ruin), where the new areas are (again, in the ruin), etc.

how's that?

**Comment by Zonespace27** [comment]
> how's that?

If I were to read this in-game, I'd have 0 idea what you've changed because you never say **what or where is actually changed in the changelog**

**Comment by Zergspower** [comment]
> 

I did?
image

**Review by Gandalf2k15 — APPROVED** [review]

**Pull request merged**